sigle/src/pages/_app.tsx

```diff
@@ -61,6 +61,17 @@ Router.events.on('routeChangeComplete', () => {
  */
 
 const GlobalStyle = createGlobalStyle`
+
+  html, body, #root, #__next{
+    height: 100%;
+  }
+
+  #__next {
+    width: 100%;
+    position: absolute;
```

Comment:
can we remove this part? doesn't seem like a really good pattern to do this on the main component.